src/Sylius/Bundle/AdminBundle/Resources/views/Layout/_logo.html.twig

```diff
@@ -1,5 +1,13 @@
-<a class="item" href="{{ path('sylius_admin_dashboard') }}" style="padding: 13px 0;">
-    <div style="max-width: 90px; margin: 0 auto;">
-        <img src="{{ asset('assets/admin/img/admin-logo.svg') }}" class="ui fluid image">
-    </div>
-</a>
+{% if use_webpack %}
```

Comment:
Just wanted to mention that you could use `set` with ternary for those, but on second thought `if`/`else` may be more readable.

Reply:
I prefer to omit `set` whenever it is possible (I mean whenever it doesn't improve readability significantly).